### ISSUE TITLE
Option::unwrap() on None in script::dom::node::iframe_browsing_context_id

### ISSUE THREAD

**Issue by mateon1** [servo/servo#18477]
Found on http://lyricsfreak.com - reproducible

```gdb
called `Option::unwrap()` on a `None` value (thread LayoutThread PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(8) }, at /checkout/src/libcore/option.rs:335)
stack backtrace:
   0:     0x55d114fb82e4 - backtrace::backtrace::libunwind::trace
                        at /shared/dev/rust/servo/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.2/src/backtrace/libunwind.rs:53
                         - backtrace::backtrace::trace<closure>
                        at /shared/dev/rust/servo/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.2/src/backtrace/mod.rs:42
   1:     0x55d114fb286f - backtrace::capture::{{impl}}::new
                        at /shared/dev/rust/servo/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.2/src/capture.rs:64
   2:     0x55d1106e5ccc - servo::main::{{closure}}
                        at /shared/dev/rust/servo/ports/servo/main.rs:130
   3:     0x55d1179cc316 - std::panicking::rust_panic_with_hook
                        at /checkout/src/libstd/panicking.rs:612
   4:     0x55d1179cc134 - std::panicking::begin_panic<alloc::string::String>
                        at /checkout/src/libstd/panicking.rs:572
   5:     0x55d1179cc0a9 - std::panicking::begin_panic_fmt
                        at /checkout/src/libstd/panicking.rs:522
   6:     0x55d1179cc03a - std::panicking::rust_begin_panic
                        at /checkout/src/libstd/panicking.rs:498
   7:     0x55d117a06660 - core::panicking::panic_fmt
                        at /checkout/src/libcore/panicking.rs:71
   8:     0x55d117a06596 - core::panicking::panic
                        at /checkout/src/libcore/panicking.rs:51
   9:     0x55d11310b817 - core::option::{{impl}}::unwrap<msg::constellation_msg::BrowsingContextId>
                        at /checkout/src/libcore/macros.rs:32
  10:     0x55d1136c7271 - script::dom::node::{{impl}}::iframe_browsing_context_id
                        at /shared/dev/rust/servo/components/script/dom/node.rs:1178
  11:     0x55d11146a5b9 - layout_thread::dom_wrapper::{{impl}}::iframe_browsing_context_id
                        at /shared/dev/rust/servo/components/layout_thread/dom_wrapper.rs:976
  12:     0x55d11128367a - layout::fragment::{{impl}}::new<layout_thread::dom_wrapper::ServoThreadSafeLayoutNode>
                        at /shared/dev/rust/servo/components/layout/fragment.rs:500
  13:     0x55d1112a5637 - layout::construct::{{impl}}::build_fragment_for_block<layout_thread::dom_wrapper::ServoThreadSafeLayoutNode>
                        at /shared/dev/rust/servo/components/layout/construct.rs:349
  14:     0x55d1112a2dca - layout::construct::{{impl}}::build_flow_for_block<layout_thread::dom_wrapper::ServoThreadSafeLayoutNode>
                        at /shared/dev/rust/servo/components/layout/construct.rs:733
  15:     0x55d1112ad680 - layout::construct::{{impl}}::build_fragment_for_absolutely_positioned_inline<layout_thread::dom_wrapper::ServoThreadSafeLayoutNode>
                        at /shared/dev/rust/servo/components/layout/construct.rs:968
  16:     0x55d1111dee23 - layout::construct::{{impl}}::process<layout_thread::dom_wrapper::ServoThreadSafeLayoutNode>
                        at /shared/dev/rust/servo/components/layout/construct.rs:1527
  17:     0x55d111177830 - layout::traversal::construct_flows_at<layout_thread::dom_wrapper::ServoLayoutNode>
                        at /shared/dev/rust/servo/components/layout/traversal.rs:205
  18:     0x55d1110f0828 - layout::traversal::{{impl}}::process_postorder<layout_thread::dom_wrapper::ServoLayoutElement>
                        at /shared/dev/rust/servo/components/layout/traversal.rs:71
  19:     0x55d11115b15f - style::traversal::DomTraversal::handle_postorder_traversal<layout::traversal::RecalcStyleAndConstructFlows,layout_thread::dom_wrapper::ServoLayoutElement>
                        at /shared/dev/rust/servo/components/style/traversal.rs:112
  20:     0x55d111264585 - style::driver::traverse_dom<layout_thread::dom_wrapper::ServoLayoutElement,layout::traversal::RecalcStyleAndConstructFlows>
                        at /shared/dev/rust/servo/components/style/driver.rs:76
  21:     0x55d11146d7b1 - layout_thread::{{impl}}::handle_reflow::{{closure}}
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:1286
  22:     0x55d1113121d8 - profile_traits::time::profile<(),closure>
                        at /shared/dev/rust/servo/components/profile_traits/time.rs:124
  23:     0x55d1110d3baf - layout_thread::{{impl}}::handle_reflow
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:1281
  24:     0x55d11146b9cd - layout_thread::{{impl}}::handle_request_helper::{{closure}}
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:699
  25:     0x55d111312918 - profile_traits::time::profile<(),closure>
                        at /shared/dev/rust/servo/components/profile_traits/time.rs:124
  26:     0x55d1110cd428 - layout_thread::{{impl}}::handle_request_helper
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:696
  27:     0x55d1110cbfcc - layout_thread::{{impl}}::handle_request
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:643
  28:     0x55d1110cb22b - layout_thread::{{impl}}::start
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:556
  29:     0x55d1111a9769 - layout_thread::{{impl}}::create::{{closure}}::{{closure}}
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:316
  30:     0x55d1110f90f2 - profile_traits::mem::{{impl}}::run_with_memory_reporting<closure,fn(profile_traits::mem::ReportsChan) -> script_layout_interface::message::Msg,script_layout_interface::message::Msg,std::sync::mpsc::Sender<script_layout_interface::message::Msg>>
                        at /shared/dev/rust/servo/components/profile_traits/mem.rs:63
  31:     0x55d1112d0741 - layout_thread::{{impl}}::create::{{closure}}
                        at /shared/dev/rust/servo/components/layout_thread/lib.rs:315
  32:     0x55d11120113a - std::sys_common::backtrace::__rust_begin_short_backtrace<closure,()>
                        at /checkout/src/libstd/sys_common/backtrace.rs:136
  33:     0x55d111202f03 - std::thread::{{impl}}::spawn::{{closure}}::{{closure}}<closure,()>
                        at /checkout/src/libstd/thread/mod.rs:394
  34:     0x55d1111aa75a - std::panic::{{impl}}::call_once<(),closure>
                        at /checkout/src/libstd/panic.rs:296
  35:     0x55d1113496a2 - std::panicking::try::do_call<std::panic::AssertUnwindSafe<closure>,()>
                        at /checkout/src/libstd/panicking.rs:480
  36:     0x55d1179d33bc - panic_unwind::__rust_maybe_catch_panic
                        at /checkout/src/libpanic_unwind/lib.rs:99
  37:     0x55d111348d8c - std::panicking::try<(),std::panic::AssertUnwindSafe<closure>>
                        at /checkout/src/libstd/panicking.rs:459
  38:     0x55d1112022b5 - std::panic::catch_unwind<std::panic::AssertUnwindSafe<closure>,()>
                        at /checkout/src/libstd/panic.rs:361
  39:     0x55d111202cfe - std::thread::{{impl}}::spawn::{{closure}}<closure,()>
                        at /checkout/src/libstd/thread/mod.rs:393
  40:     0x55d111251193 - alloc::boxed::{{impl}}::call_box<(),closure>
                        at /checkout/src/liballoc/boxed.rs:682
  41:     0x55d1179caf9b - alloc::boxed::{{impl}}::call_once<(),()>
                        at /checkout/src/liballoc/boxed.rs:692
                         - std::sys_common::thread::start_thread
                        at /checkout/src/libstd/sys_common/thread.rs:21
                         - std::sys::imp::thread::{{impl}}::new::thread_start
                        at /checkout/src/libstd/sys/unix/thread.rs:84
  42:     0x7f3ff020d493 - start_thread
  43:     0x7f3fefd3aabe - __clone
  44:                0x0 - <unknown>
ERROR:servo: called `Option::unwrap()` on a `None` value
```

**Comment by jdm** [comment]
This looks like the same issue as #17482.

**Comment by atouchet** [comment]
Was this fixed in #18514?

**Comment by asajeffrey** [comment]
Yes.
